src/k8s-engine/lima.ts

```diff
@@ -575,7 +571,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     return this.status.then(defined);
   }
 
-  private calcRandomTag(desiredLength: number) {
+  static calcRandomTag(desiredLength: number) {
```

Comment:
```suggestion
  private static calcRandomTag(desiredLength: number) {
```

There's no reason this needs to be public.